src/config/dynamic-theme-fixes.config

```diff
@@ -291,6 +291,20 @@ INVERT
 
 ================================
 
+br.de
+
+INVERT
+article h4
```

Comment:
This website has a strange behavior, it tries to remove Dark Reader's style sheets. For some reason they were restored not properly.

Inversion in Dynamic mode should be used for images, but not texts (with the exception of closed Web Components, like Google Translate extension's popup).

So, let's apply a CSS rule like `color: ${#323232} !important;`.

Reply:
Reworked for full CSS fix.

Not so nice but working.